macosx/PrefsController.m

```diff
@@ -1015,8 +1015,8 @@ - (void) updateRPCPassword
 
         tr_sessionSetRPCPassword(fHandle, fullPassword);
 
-        fRPCPassword = [[NSString alloc] initWithUTF8String: fullPassword];
-        [fRPCPasswordField setStringValue: fRPCPassword];
+        fRPCPassword = @(fullPassword);
```

Comment:
Previously retained, now autoreleased. Not sure if this is the only place.

Reply:
Fixed here and everywhere